common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/SmartcardNfcLoadingDialog.java

```diff
@@ -68,10 +68,11 @@ public void run() {
     }
 
     /**
-     * Should dismiss dialog and call the appropriate methods to help cancel the CBA flow.
+     * Called when smartcard is unexpectedly disconnected via USB from device.
+     * Used to run any cancellation logic needed (without the cancel button needing to be pressed).
      */
     @Override
-    void onCancelCba() {
-        //No options to cancel here.
+    void onUnexpectedUnplug() {
+        //This method is for NFC, so it will never be called from here.
```

Comment:
Should this method not be here? Not sure I understand the comment here

Reply:
Same for the similar comments below

Reply:
The onUnexpectedUnplug() method is a member of the SmartcardDialog abstract class, so it needs to be implemented in all of the children. This dialog and a few others are meant to be shown in the NFC flow only, so there shouldn't be any scenario where this method is being called from this particular dialog.
I would have created an abstract subclass meant for USB dialogs and put this onUnexepectedUnplug() method there, but there aren't any USB specific dialogs (they're shared between USB and NFC). 
Some sort of refactoring for this is planned for a future PR.